Firestore/Source/Core/FSTQuery.mm

```diff
@@ -735,7 +479,7 @@ - (nullable const FieldPath *)firstSortOrderField {
 }
```

Comment:
Can this be `return _query.FirstSortOrderField`?

Reply:
It can once it's implemented :-). This PR does not include order-by so `FSTQuery` is still responsible for those.